guides/ssl-certificates/azureDNS.md

```diff
@@ -31,7 +31,7 @@ are the same regardless of which option you choose.
 
 You must have:
 
-- A Kubernetes cluster with internet connectivity
+- A Kubernetes cluster [of a supported version](https://kubernetes.io/releases/version-skew-policy/#supported-version-skew) with internet connectivity
```

Comment:
FYI: now that https://github.com/cdr/docs/pull/232 was merged, you can link to that page instead https://coder.com/docs/coder/v1.20/setup/kubernetes#supported-kubernetes-versions